### PR TITLE
Fix Godeps

### PR DESCRIPTION
**Pull request by eparis**
Godeps had two problems:
1. godep stopped including license files in recursive godeps.
2. google.golang.org/api/storage/v1beta1 disappeared so move to the v1 API

### CODE REVIEW

**Comment by eparis** [comment]
@brendandburns @rootfs @quinton-hoole we'll see if this one passes travis or not.

**Comment by eparis** [comment]
This is solving the same problem as both #17478 and #17457

**Comment by eparis** [comment]
CI green godeps, wohoo!

**Comment by rootfs** [comment]
@eparis  you rock!

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
 			"Comment": "null-5",
-			"Rev": "75cd24fc2f2c2a2088577d12123ddee5f54e0675"
+			"Rev": "'75cd24fc2f2c2a2088577d12123ddee5f54e0675'"
```
Comment (mikedanese) [inline Godeps/Godeps.json]: weird extra apostrophes?

Comment (ncdc) [inline Godeps/Godeps.json]: +1, what's up with that?

Comment (eparis) [inline Godeps/Godeps.json]: couldn't say, it showed up in my, brendan's and huamin's update. I'm clueless why, but it works.

**Comment by ncdc** [comment]
The extra apostrophes are weird, but no objection if everything is green. LGTM

**Comment by brendandburns** [comment]
Thanks @eparis

**Pull request merged**